experiments/jz/joint_training_toy/joint_training_toy1/04_do_training.slurm

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
-#SBATCH --job-name=modelling-metadata-html-XX                          # (change me!) job name
+#SBATCH --job-name=modelling-metadata-html-joint-toy-1
 #SBATCH --ntasks=1                    # number of MP tasks
 #SBATCH --constraint=v100-16g
 #SBATCH --cpus-per-task=8                                              # (change me! between 0 and 40) number of cores per tasks
 #SBATCH --hint=nomultithread                                           # we get physical cores not logical
 #SBATCH --time 01:00:00                                                # (change me! between 0 and 20h) maximum execution time (HH:MM:SS) 
+#SBATCH --gres=gpu:1                                                    # (change me! between 0 and 1) number of gpus 
```

Comment:
Just to be sure, we didn't want to test the parallel data training feature here? :relaxed:

Reply:
I just read the comment on slack, it's fine for me if we test it later!